plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeMetadata.java

```diff
@@ -465,6 +473,35 @@ public void testGetInputInfoForUnPartitionedTable()
         assertThat(deltaLakeMetadata.getInfo(tableHandle)).isEqualTo(Optional.of(new DeltaLakeInputInfo(false)));
     }
 
+    @Test
+    public void testCreateSchemaTwiceInTheSameSession()
```

Comment:
This feels like a unit test and somebody without a context for the `SchemaAlreadyExistsException` problem would have a hard time to figure out what are the tests there.
What about using a specific test - similar to `TestIcebergFileMetastoreTableOperationsInsertFailure` ?